packages/ti_anomali/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.26.3"
```

```suggestion
- version: "1.27.0"
```
enhancement bump minor